s4e-web/src/app/views/map-view/state/map/map.query.ts

```diff
@@ -28,7 +29,8 @@ export class MapQuery extends Query<MapState> {
         overlays: overlays.length === 0 ? null : overlays,
         product: product == null ? null : product.active,
         scene: scene == null ? null : scene.id,
-        date: product.ui.selectedDate,
+        date: !!scene && moment(scene.timestamp).format('YYYY-MM-DD[T]HH:mm:ss[Z]')
+          || moment(product.ui.selectedDate).format('YYYY-MM-DD'),
```

Comment:
Shouldn't a date be always in `YYYY-MM-DD` format? After all `YYYY-MM-DD[T]HH:mm:ss[Z]` is not a date anymore, but a timestamp/date-time.
I don't mean that it's erroneous but I'm trying to understand why would that make sense, as the original was of setting it by means of `product.ui.selectedDate` seems more sensible to me at the moment.